text/03.md

```diff
@@ -707,6 +831,19 @@ JSON object with the properties `as` (can be `lawyer` or `writer`), `action`,
 and `document`. We described the format in the first `system` message, but
 showing a couple of examples helps.
 
+**ID> This example was very hard for me to understand ...
+
+The nested JSON was confusing...
+
+An alternative is to build the output as a format string, instanitating the templates as input...
```

Comment:
That was my initial approach but that also becomes a mess in other situations where you have to escape { and } for format to work properly. Intentionally switched to handlebar parameters at my colleague Ransom's suggestion, which made things overall better.